docs/source/model_design/vivarium_model_components/GBD_disease_health/index.rst

```diff
@@ -30,14 +30,137 @@ Understanding GBD's Disease and Health Categories
+a cause. For example, IHD has medical conditions like acute myocardial infarction 
+(a heart attack) and angina (chest pain) as associated sequelae. These are the things 
+a patient might be experiencing, or a doctor might identify. 
+
+Sequelae often contain a level of severity for that condition - for example, mild and moderate 
+angina are both sequelae within IHD. 
+
+Health States
++++++++++++++
+
+A health state is the amount of health or disability within a sequelae. This is summarized 
+through the disability weight. More information on this can be found in `this section of the cause model page <https://vivarium-research.readthedocs.io/en/latest/model_design/vivarium_model_components/causes/index.html#disability-weights>`_. A health state for a sequelae and its disability weight can be 
+found in the GBD methods appendix. 
+
+Using our above sequelae (acute myocardial infarction and angina) as an example - 
+the amount of health lost while experiencing an acute 
+myocardial infarction is quite high! You're likely hospitalized and in a lot of pain. 
+However, the health lost from angina might be less significant. You're likely at home and 
+might only experience discomfort when under stress. This affects your disability weight value. 
+
+Each sequela has exactly one health state associated with it. If you are experiencing multiple 
+symptoms, such as "headache and chest pain", your health state contains all of these losses to 
+health in one "health state". 
+
+One important note is that health states can be shared across sequelae - and even shared 
+between sequelae of different causes. For example, severe heart failure due to IHD and 
+severe heart failure due to HHD have the same amount of health loss associated with them. 
+Severe heart failure feels the same to a patient no matter the cause. Therefore, we would 
+say that the health state and therefore the disability weight is the same for both sequelae. 
+
+Impairments
++++++++++++
+
+Sometimes, GBD has a lot of information about a health state - such as severe heart failure in the 
+example above - that the modelers want to utilize. However, in the process of normal GBD 
+modeling, this wouldn't be possible. Usually, the cause level information is calculated 
+first and then divded into sequelae. 
+
+Therefore, to use this valuable information about a health state, GBD created a new 
+category called impairments. An impairment is medically the same as a sequela, but 
+there is more information about it. To include this, GBD modelers first find the total 
+amount of the impairment - heart failure - and then divide it between causes. This 
+creates the sequelae of "heart failure due to IHD" and "heart failure 
+due to HHD" for example. These are included in blue in the diagram above. 
+
+These sequelae therfore have the same health state. Therefore, an 
+impairment is a grouping of the related health states (mild/moderate/severe heart failure for 
```

Comment:
I'm not sure it's accurate to think of an impairment as a grouping of related health states, because some sequelae (and hence health states) for a given cause are the result of combining _multiple_ impairments. The example given in the powerpoint is
> "Severe motor plus cognitive impairment with blindness and epilepsy due to neonatal preterm birth complications 32-<37wks," 

which involves _three_ impairments (blindness, epilepsy, and intellectual disability) due to one cause (neonatal preterm birth complications 32-<37wks). So the health state for this particular sequela is the result of combining three different impairments. The image Ali posted above about chronic kidney disease might be another example, but I'm not clear on what's a cause vs. impairment there.

So I guess we _could_ say that the impairment corresponds to the the individual health states we'd get in the absence of other impairments, but that seems like a confusing way to think about it when impairments can get combined, resulting in brand new health states.

Reply:
> we could say that the impairment corresponds to the the individual health states we'd get in the absence of other impairments

I think that is what we're saying -- that the vision loss impairment includes the health states of "vision loss with epilepsy and intellectual disability," "vision loss with epilepsy," "vision loss with intellectual disability," etc. One of the key features of impairments (see my comment about the cause hierarchy below) is that they are not mutually exclusive nor collectively exhaustive. Perhaps I'm missing something wrong with this framing.